workshops/fabric-e2e-rag/workshop.md

```diff
@@ -0,0 +1,480 @@
+short_title: Building RAG Application on Microsoft Fabric & Azure Open AI
+description: This workshop will guide you through the process of building a Retrieval augmented generation (RAG) application on Microsoft Fabric and Azure Open AI.
+level: beginner
+authors:
+  - Josh Ndemenge
+contacts:
+  - '@Jcardif'
+duration_minutes: 60
+tags: data, Microsoft Fabric, Azure Open AI
+banner_url: assets/architecture.png
+sections_title:
+  - Introduction
+  - Loading and Preprocessing PDF Documents
+  - Generating Embeddings and Storing them in a Vector Store
+  - Retrieving Relevant Documents and Answering Questions
+  - Conclusion
+
+wt_id: data-114676-jndemenge
+
+---
+
+# Introduction
+
+In this workshop, we'll demonstrate how to develop a context-aware question answering framework for any form of a document using [OpenAI models](https://azure.microsoft.com/products/ai-services/openai-service), [SynapseML](https://microsoft.github.io/SynapseML/) and [Azure AI Services](https://azure.microsoft.com/products/cognitive-services/). The source of data for this workshop is a PDF document, however, the same framework can be easily extended to other document formats too.
+
+
+We’ll cover the following key steps:
+
+1. Preprocessing PDF Documents: Learn how to load the PDF documents into a Spark DataFrame, read the documents using the [Azure AI Document Intelligence](https://azure.microsoft.com/products/ai-services/ai-document-intelligence) in Azure AI Services, and use SynapseML to split the documents into chunks.
+2. Embedding Generation and Storage: Learn how to generate embeddings for the chunks using SynapseML and [Azure OpenAI Services](https://azure.microsoft.com/products/cognitive-services/openai-service), store the embeddings in a vector store using [Azure Cognitive Search](https://azure.microsoft.com/products/search), and search the vector store to answer the user’s question.
+3. Question Answering Pipeline: Learn how to retrieve relevant document based on the user’s question and provide the answer using [Langchain](https://python.langchain.com/en/latest/index.html#).
+
+To get an indepth understanding of the RAG framework, refer to [this workshop](https://moaw.dev/workshop/gh:azure-samples/azure-openai-rag-workshop/base/docs/)
+
+## Prerequisites
+
+To follow along with this work begin by Right-click or `Ctrl + Click` on the button below to open the Azure Portal in a new tab and begin deployment of the necessary resources.
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2moaw%2Fmain.json)
+
+This deploys the following resources:
+1. Azure OpenAI Service
+2. Azure AI Search
+3. Azure AI Services
+4. Azure Key Vault
+
+<div class="info" data-title="Note">
+  
+  > The Azure Open AI deployment also includes the deployment of the ***gpt-4*** and ***text-embedding-ada-002 models*** 
```

Comment:
`text-embedding-ada-002` is already included in Azure AI Services for Fabric. Is there a way to use it instead of deploying it on Azure AI Services?